src/components/HealthFacilityFilter.js

```diff
@@ -246,6 +247,16 @@ class HealthFacilityFilter extends Component {
             }
           />
         </Grid>
+        {this.isHealthFacilityStatusEnabled && <Grid item xs={3} className={classes.item}>
+          <PublishedComponent
+            module="location"
+            label="HealthFacilityForm.status"
+            pubRef="location.HealthFacilityStatusPicker"
+            value={this._filterValue("status")}
+            onChange={(v, s) => this._onChange("status", v, s)}
```

Comment:
Use more explicit name for 'v'. I believe 's' argument is not necessary here.

Reply:
I would like to omit the "s" part here, I would have to do a little refactor to the whole component. I want to keep things separate between PR's.